src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,78 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onChangePage: (page: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onChangePage,
+}) => {
+  const pagesAmount = Math.ceil(total / perPage);
+  const pages = [];
+
+  for (let i = 1; i <= pagesAmount; i += 1) {
+    pages.push(i);
+  }
```

Comment:
You shouldn't use loops outside of the function or hook in the component. 
Consider creating a full array instantly